chunks/lang-java/Dockerfile

```diff
@@ -2,6 +2,7 @@ ARG base
 FROM ${base}
 
 ARG JAVA_VERSION
+ARG JAVA_VERSION_SECONDARY
```

Comment:
Leftover?

Reply:
Oh yes, thanks for noting. I removed it as well, but forgot to push 😭

Reply:
https://github.com/gitpod-io/workspace-images/pull/888/commits/946522bde06dad4ffa7eed7ca01c4a2d99fbd458